kolibri/core/assets/src/state/actions.js

```diff
@@ -200,10 +200,15 @@ function kolibriLogout(store) {
   }).catch(error => { handleApiError(store, error); });
 }
 
-function getCurrentSession(store) {
+function getCurrentSession(store, force = false) {
```

Comment:
I'm not sure. There are two common ways of updating the Session after such an update. First, you can get the fresh data via a force refetch from the backend like you've done here. Or you could 'optimistically' update the Session by applying the update data directly to the client-side Session (and save yourself a call to the Session API). I think the first route is probably the more conservative approach.

My question is, if you go the second route, whether the Session Resource class will have stale, not-updated, data the next time you load a page that requires Session data because of how its cached.